yb-voyager/cmd/exportData.go

```diff
@@ -177,6 +177,17 @@ func exportData() bool {
 		os.Exit(0)
 	}
 
+	finalTableList, err = addLeafPartitionsInTableList(finalTableList)
+	if err != nil {
+		utils.ErrExit("failed to add the leaf partitions in table list: %w", err)
+	}
+
+	metaDB.UpdateMigrationStatusRecord(func(record *metadb.MigrationStatusRecord) {
+		record.RenameTablesMap = strings.Join(lo.MapToSlice(partitionsToRootTableMap, func(k, v string) string {
```

Comment:
why stringify it? We should be okay to store it as a map itself, right? You wouldn't need the logic in `getRenameTableMap` then.